dev/NavigationView/NavigationView_rs1_themeresources.xaml

```diff
@@ -61,9 +61,9 @@
 
             <StaticResource x:Key="NavigationViewSelectionIndicatorForeground" ResourceKey="AccentFillColorDefaultBrush" />
 
-            <StaticResource x:Key="TopNavigationViewItemForeground" ResourceKey="TextFillColorSecondaryBrush" />
+            <StaticResource x:Key="TopNavigationViewItemForeground" ResourceKey="TextFillColorPrimaryBrush" />
```

Comment:
The brushes for TopNavigationViewItems are correct. This can be reverted.

Reply:
waiting until design responds